### ISSUE TITLE
Wrong parameter in CopyleaksProcess invocation

### ISSUE THREAD

**Issue by gioaudino** [Copyleaks/Python-Plagiarism-Checker#7]
The invocation of the `CopyleaksProcess` constructor in the `getProcesses()` method raises an Exception: `AttributeError: 'function' object has no attribute 'generateAuthrizationHeader'`.
The second parameter should be `self.token` and not `self.login`

**Comment by gioaudino** [comment]
`self.login` in CopyleaksCloud is a method of the class. It gets passed to the method `parseProcesses` where it's named `token` (see [copyleakscloud.py](https://github.com/Copyleaks/Python-Plagiarism-Checker/blob/master/copyleaks/copyleakscloud.py) line 95) and from there passed to the `CopyleaksProcess` constructor as the second parameter (see [copyleaksprocess.py](https://github.com/Copyleaks/Python-Plagiarism-Checker/blob/master/copyleaks/copyleaksprocess.py) line 167). There the parameter is named `loginToken` and gets stored into the field `token`.  
When invoking the method `getResult` on such `CopyleaksProcess` object, the method `generateAuthrizationHeader` is invoked on the field token that, as above, is a function and not an the token object, thus raising the Exception. An example of the code could be:
```
self._cloud = CopyleaksCloud(Product.Websites, "email", "apikey")
scans = self._cloud.getProcesses()
results = [process.getResults() for process in scans]
```

**Comment by gioaudino** [comment]
@erezleaks of course there's no problem if getProcesses returns an empty list

**Comment by erezleaks** [comment]
@gioaudino your changes have been merged and published in copyleaks-2.5.3
We appreciate your assistance
